missions/W4/M2/Dockerfile

```
@@ -0,0 +1,89 @@
+WORKDIR /home/hduser
+USER hduser
+# 
+# RUN ssh-keygen -t rsa -P '' -f ~/.ssh/id_rsa && cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys && chmod 0600 ~/.ssh/authorized_keys
+ENV HADOOP_VERSION=3.3.3
+ENV HADOOP_HOME=/home/hduser/hadoop-${HADOOP_VERSION}
+
+ENV SPARK_VERSION=3.5.4
+ENV SPARK_HOME=/home/hduser/spark-${SPARK_VERSION}
+
+# 모든 셸에서 환경변수를 사용할 수 있도록 설정
+RUN echo "export HADOOP_VERSION=${HADOOP_VERSION}" | sudo tee -a /etc/profile.d/hadoop.sh && \
```

Hadolint (2.12.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)